### ISSUE TITLE
Gemfiles with multiple sources don't work

### ISSUE THREAD

**Issue by tylerholien** [aflatter/bundix#2]
I've tried with just the bare sources at the top of the Gemfile, with source blocks (e.g. `source 'https://my-gemserver' do ...`), and with the `:source` attribute on individual gems. I can only ever get it to use rubygems.

**Comment by aflatter** [comment]
@tylerholien I haven't looked at bundix for quite some time and still have to look at the changes @cstrahan has been doing in nixpkgs and his fork of bundix.

**Comment by cstrahan** [comment]
@tylerholien Yeah, I didn't have any non-rubygems.org gems to test with at the time, so I need to dive back in and add support for that, which I hope to do in the next couple days.

**Comment by tylerholien** [comment]
@cstrahan Awesome, thanks!

**Comment by aflatter** [comment]
@cstrahan You can use `gem 'rails-assets-bootstrap', source: 'https://rails-assets.org'` :)

**Comment by Denommus** [comment]
@cstrahan diaspora might be a good use-case for a project that uses other sources.
